apps/oozie/src/oozie/tests.py

```diff
@@ -387,7 +387,7 @@ def create_noop_workflow(self, name='noop-test'):
 
     wf = Workflow.objects.new_workflow(self.user)
     wf.name = name
-    wf.save()
+    # wf.save()
```

Comment:
After commenting out wf.save() this is not giving error (random checking). Just checking what is the output and overall thing for the test failure in py3. There are 2 files for model in oozie. And trying to understanding the workflow object here. So need the birds eye view for workflow object.
failed test link -> https://app.circleci.com/pipelines/github/cloudera/hue/4490/workflows/315d35f7-d1c2-4b51-90c1-b3f066e3837e/jobs/6156
passed test link -> https://app.circleci.com/pipelines/github/cloudera/hue/4493/workflows/217df8fa-fc53-4a9e-a0a3-0591221a4739/jobs/6162

Reply:
This is very old code (oozie.models.py Models have been replaced by Document2), so fine to just "make it work" or start skipping the tests.